bedrock/contentful/management/commands/update_contentful.py

```diff
@@ -171,13 +171,20 @@ def _queue_has_viable_messages(self) -> bool:
             ACTION_UNPUBLISH,
             ACTION_DELETE,
         }
-        EXTRA_DEV_GO_ACTIONS = {
+        EXTRA_PREVIEW_API_GO_ACTIONS = {
+            # Sites that are using the preview API key (Dev and Stage) act upon
+            # two more state changes
             ACTION_AUTO_SAVE,
             ACTION_SAVE,
         }
 
-        if settings.DEV:
-            GO_ACTIONS = GO_ACTIONS.union(EXTRA_DEV_GO_ACTIONS)
+        if settings.APP_NAME in [
```

Comment:
might be easier and more reliable to do this when not prod instead? i.e. `if settings.APP_NAME != "bedrock-prod":`

Reply:
Yeah, i don't mind going that way instead. Looking at it now I think my intent was to be explicit about where it does run, rather than doesn't, but that depends too much on knowing that `bedrock` is the default app name.  

So yeah, will amend pre-merge 👍

Reply:
I'm not sure either, and it's probably going to be effectively the same. If you like this better it's fine with me, just wanted to ask to be sure.